docs/src/test-reporters-js.md

```diff
@@ -511,24 +543,36 @@ The following test adds attachments:
 
 ```js tab=js-js
 // @ts-check
-const { test } = require('@playwright/test');
-
-test('embed attachments, including its content, on the JUnit report', async ({}, testInfo) => {
-  const file = testInfo.outputPath('evidence1.txt');
-  require('fs').writeFileSync(file, 'hello', 'utf8');
-  await testInfo.attach('evidence1.txt', { path: file, contentType: 'text/plain' });
-  await testInfo.attach('evidence2.txt', { body: Buffer.from('world'), contentType: 'text/plain' });
+const { test } = require("@playwright/test");
+
+test("embed attachments, including its content, on the JUnit report", async ({}, testInfo) => {
+  const file = testInfo.outputPath("evidence1.txt");
+  require("fs").writeFileSync(file, "hello", "utf8");
+  await testInfo.attach("evidence1.txt", {
```

Comment:
Maybe the more compact code style that was originally used is better for docs ? 🤔
```
await testInfo.attach('evidence1.txt', { path: file, contentType: 'text/plain' });
```